crates/dbsp/src/circuit/runtime.rs

```diff
@@ -109,13 +109,15 @@ mod thread_type {
     #[cfg(doc)]
     use super::Runtime;
     use enum_map::Enum;
+    use serde::Serialize;
 
     thread_local! {
         static CURRENT: Cell<ThreadType> = const { Cell::new(ThreadType::Foreground) };
     }
 
     /// Type of a thread running in a [Runtime].
-    #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash, Enum)]
+    #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash, Enum, Serialize)]
+    #[serde(rename_all = "lowercase")]
```

Comment:
Is it confusing that we seem to use `rename_all = "snake_case"` everywhere else in our code-base?